app/controllers/api/account.php

```diff
@@ -2503,15 +2507,18 @@
             'hello' => $locale->getText("emails.recovery.hello"),
             'name' => $profile->getAttribute('name'),
```

Comment:
Do we still need `name` if we updated it to `user`?

Reply:
Not necessarily. I can refactor the mails worker to not use `name` anymore 👍